src/index.html

```diff
@@ -5,8 +5,347 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        src="./images/logo.png"
+        alt="logo"
+        class="logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link" href="#laptops"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#gadjets">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="content">
+      <div class="content__container">
+        <section class="section" data-qa="card">
+          <div class="section__item product">
+            <div class="product__image">
+              <img
+                src="./images/imac.jpeg"
+                alt="logo"
+                class="product__photo"
+              >
+            </div>
+            <div class="product__wrap">
+              <h3 class="product__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
```

Comment:
Тут теж раджу текст починати з нового рядка
```suggestion
              <h3 class="product__title">
                Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
              </h3>
```

Reply:
done